modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectDefinition.tsx

```diff
@@ -47,21 +47,38 @@ const ModalAddObjectDefinition: React.FC<IProps> = ({
 	apiURL,
 	observer,
 	onClose,
-	storageTypes,
+	storages,
 }) => {
+	const storageSortedByLabel = [...storages].sort(
```

Comment:
@igor-franca This will sort only the first two options, but it will be ok if the client extensions options come after those two. @MarinhoFeliphe could you confirm if the new options will be added at the end of the `storages` array?